bc_obps/reporting/tests/service/test_report_activity_save_service/infrastructure.py

```diff
@@ -45,14 +44,12 @@ def build(cls):
         t = TestInfrastructure()
         t.facility_report = make_recipe(
             'reporting.tests.utils.facility_report',
-            report_version__report__created_at=datetime(1999, 9, 1, tzinfo=timezone.utc),
         )
+        t.facility_report.refresh_from_db()
         t.report_version = t.facility_report.report_version
         t.user = make_recipe('registration.tests.utils.industry_operator_user')
-        t.configuration = make_recipe(
-            "reporting.tests.utils.configuration",
-            valid_from=datetime(1999, 1, 1, tzinfo=timezone.utc),
-            valid_to=datetime(1999, 12, 31, tzinfo=timezone.utc),
+        t.configuration = Configuration.objects.get(
+            valid_from__lte=t.facility_report.created_at, valid_to__gte=t.facility_report.created_at
```

Comment:
I'm not a big fan of tests that depend on what time it is when they run...But as things are right now I don't think it's worth getting into mocking database time for that